authors/settings/base.py

```diff
@@ -84,18 +85,27 @@
 # Password validation
 # https://docs.djangoproject.com/en/1.11/ref/settings/#auth-password-validators
 
+USER_ATTRIBUTE_SIMILARITY_VALIDATOR = \
+    'django.contrib.auth.password_validation.UserAttributeSimilarityValidator'
+MINIMUM_LENGTH_VALIDATOR = \
+    'django.contrib.auth.password_validation.MinimumLengthValidator'
+COMMON_PASSWORD_VALIDATOR = \
+    'django.contrib.auth.password_validation.CommonPasswordValidator'
+NUMERIC_PASSWORD_VALIDATOR = \
+    'django.contrib.auth.password_validation.NumericPasswordValidator'
+
 AUTH_PASSWORD_VALIDATORS = [
     {
-        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+        'NAME': USER_ATTRIBUTE_SIMILARITY_VALIDATOR,
     },
     {
-        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+        'NAME': MINIMUM_LENGTH_VALIDATOR,
     },
     {
-        'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
+        'NAME': COMMON_PASSWORD_VALIDATOR,
```

Comment:
@oma0256, good work here. Thanks. I see you have made some changes in the settings file. Do these two lines mean the same thing or some env variables need to be set in the env file?

Reply:
@pbkabali Yea they do mean the same thing I was just trying to avoid pep8 issues

Reply:
OK cool. Thanks